vault/dendron.topic.links.md

```diff
@@ -120,6 +120,6 @@ Markdown links can be used for local files and stuff on the internet (websites,
 
 ### Convert Link
 
-![[dendron://dendron.dendron-site/dendron.topic.refactoring#convert-link,1:#*]]
+![[dendron://dendron.dendron-site/dendron.topic.refactoring.commands.convert-link#summary,1:#*]]
```

Comment:
this raises a good question of how to handle commands/features that fall into multiple categories. convert link can fit into both `refactoring` as well as `links`. maybe this is a good argument for mastering all commands horizontally into the `ref.*` hierarchy so that we don't end up with this dilemma 🤔

Reply:
My impression from writing multiple documents for both concrete commands and conceptual topics is that any documentation for a concrete implementation of a feature that you can physically point at in our codebase benefits from canonically being documented in a horizontal fashion (in `ref`).

It is rare that any piece of contributed feature in our extension is going to fall strictly under one conceptual topic. Even if it does, Dendron will continue to evolve and many features will fall under multiple conceptual topics eventually.

Like you suggested, the first most immediate candidate for this are the commands. I can imagine we will be hitting this dilemma in other contribution points as well.

How about we take a slight pivot from our stance on topic documentation as following?:

- The content that pertains to a higher order conceptual explanation remains to be written vertically under `topic`s.
  - `topics` is our canonical location for high level concepts
- Atomic units of functionality that exists concretely in a contained boundary of our codebase are written horizontally under `refs`. (currently, only `commands`).
  - `ref` is our canoncial location for concrete units of functionality.
- `topic`s transclude notes in `ref` to explain concepts.
- `ref`s would most likely not transclude anything from `topic`s.
- The sum of `topic`s and `ref`s is our _source of truth_.

Reply:
Lets experiment. For commands, author them primarly in ref. for everything else, lets keep the topic convention. can you update `[[References|dendron://dendron.handbook/leaflet.journal.2022.05.10.standard-documentation.references]]` to reflect this? also add instructions on what we'd need to do to disambiguate (eg. note-lookup vs schema lookup command, how will those be structured?)

Reply:
leaving this as future work as per discussion.